opm/parser/eclipse/EclipseState/Tables/SimpleTable.hpp

```diff
@@ -75,18 +56,11 @@ namespace Opm {
         double evaluate(const std::string& columnName, double xPos) const;
 
     protected:
-        void checkNonDefaultable(const std::string& columnName);
-        void checkMonotonic(const std::string& columnName,
-                             bool isAscending,
-                             bool isStrictlyMonotonic = true);
-        void assertUnitRange(const std::string& columnName);
-        void applyDefaultsConstant(const std::string& columnName, double value);
-        void applyDefaultsLinear(const std::string& columnName);
-        void createColumns(const std::vector<std::string> &columnNames);
-
         std::map<std::string, size_t> m_columnNames;
-        std::vector<std::vector<double> > m_columns;
         std::vector<std::vector<bool> > m_valueDefaulted;
+        std::vector<TableColumn> m_columns2;
```

Comment:
(the point is that the name is useless. also instead of having two copies of every column, the values of `m_columns` should point to the objects in `m_columns2` because `m_columns` is just an alias, right?)